dp/pipeline/epc_arp.c

```diff
@@ -84,7 +84,7 @@
 /**
  * no. of mbuf.
  */
-#define NB_ARP_MBUF  1024
+#define NB_ARP_MBUF  4096//1024
```

Comment:
This change is not specific to AF_PACKET. Correct? Can we handle it in separate PR with reasoning and what is memory requirement change?

Reply:
I undid this change. I need further stress testing before suggesting this change.